docs/docs/policies.mdx

```diff
@@ -176,6 +176,24 @@ If you want to fine-tune your model, start by modifying the following parameters
   set `weight_sparsity` to 1 as this would result in all kernel weights being 0, i.e. the model is not able
   to learn.
 
+* `split_entities_by_comma`:
```

Comment:
please add this param to the param table as well

Reply:
Could you add an example for an address in addition to the `banana,apple` example in order to illustrate when splitting an entity does not make sense?